objects/src/notes/metadata.rs

```diff
@@ -22,24 +22,27 @@ pub enum NoteExecutionMode {
 // NOTE METADATA
 // ================================================================================================
 
-/// Represents metadata associated with a note.
-///
-/// The metadata consists of:
-/// - sender is the ID of the account which created the note.
-/// - note_type defines how the note is to be stored (e.g., on-chain or off-chain).
-/// - tag is a value which can be used by the recipient(s) to identify notes intended for them.
-/// - aux is arbitrary user-defined value.
```

Comment:
Moved the documentation to the fields. Having the comments along side thing they refer to makes it easier to update when changes are done.

Reply:
I don't think that is a problem. If the attribute is private, the docs should not be shown because it is not part of the public API. The user has to rely on the public API, in this case the accessor methods (they have documentation).